### DISCUSSION TITLE
Log description

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [AdguardTeam/AdGuardHome#3081] [Q&A]
Hello,

I try to get the meaning of each field but i would like to get the right meaning of each please :)

IP : client IP
T : for Time
QH : query host?
QT : ? Query Type?
QC : ?
CP : ?
Answer : ?? (i always got a something like lHiBgAABAAEAAAAABHd3dz...., any chance to get it in clear?)
Result : ?
Elaspsed : time for the query to be resolve?
Upsteam : the dns where the request is  forward

I miss this field : 
- message-type : Query or Response, where ?  (looking for Type of DNS message.)

Thank you :)

**Comment by ameshkov** [comment]
It's all in the code:)

https://github.com/AdguardTeam/AdGuardHome/blob/master/internal/querylog/decode.go#L19

**Reply by b4b857f6ee** [reply]
Ok thank you, i will check.
Can we remove the base64 encoding? (UI option i miss maybe?)

**Reply by ameshkov** [reply]
Not really, DNS answer is binary, you can't store it as is

**Reply by b4b857f6ee** [reply]
ok thank you :).
I will rebuild my parser for Splunk with this information :).
